src/Entries/EntryQueryBuilder.php

```diff
@@ -62,6 +62,10 @@ public function find($id, $columns = ['*'])
 
     public function get($columns = ['*'])
     {
+        if ($columns !== ['*']) {
+            $columns = array_unique(array_merge($columns, ['site', 'origin_id', 'slug', 'collection', 'created_at', 'updated_at']));
```

Comment:
It would be better to merge with `self::COLUMNS` to avoid repetition? But really with this change youre just getting all the columns anyway.